server/routes/api/v1/pod/create.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (express) {
 
     // Example of data in JSON format
     // pod.add({current_weight: 124, max_weight: 900, last_maintain: 1200}
-    pod.create({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
+    pod.add({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
```

Comment:
pod.create should be the notation not .add. Please pull master to make sure that your model.js reflect an create function return not an add